docs/developer-guide.rst

```diff
@@ -1,82 +1,84 @@
+.. _developer-guide:
+
 Set up development environment
 ------------------------------
 
-This page describes how to set up the development environment for
-working on ``c2cgeoform``. It is for developers working on
-``c2cgeoform`` itself, not for developers working on
-``c2cgeoform``-based applications.
+This page describes how to set up the development environment for working on
+c2cgeoform. It is for developers working on c2cgeoform itself, not for
+developers working on c2cgeoform-based applications.
+
+Note that c2cgeoform is a framework and with a
```

Comment:
???

Reply:
Note that c2cgeoform is a framework  with a pyramid scaffold....... used to create ....